src/org/labkey/test/util/Crawler.java

```diff
@@ -226,22 +226,31 @@ public Crawler(BaseWebDriverTest test, Collection<String> projects, Duration cra
             new ControllerActionId("specimen", "downloadSpecimenList"),
             new ControllerActionId("specimen", "emailLabSpecimenLists"),
             new ControllerActionId("specimen", "getSpecimenExcel"),
+
+            // We're in the process of moving actions from the "specimen" controller in the study module to the
+            // "specimen2" controller in the specimen module. For now, exclude both sets. After migration is complete,
+            // eliminate the "specimen2" variants.
+            new ControllerActionId("specimen2", "download"),
+            new ControllerActionId("specimen2", "downloadSpecimenList"),
+            new ControllerActionId("specimen2", "emailLabSpecimenLists"),
+            new ControllerActionId("specimen2", "getSpecimenExcel"),
```

Comment:
If you want to skip the "specimen2" actions entirely, you could add it to `getExcludedControllers`.

Reply:
We want most of them to be tested; just excluding the same problematic ones as they shift to "specimen2" and then back to "specimen"